src/app/components/Item/ItemTableRow.jsx

```diff
@@ -48,11 +50,14 @@ class ItemTableRow extends React.Component {
       bibId,
       searchKeywords,
     } = this.props;
-    const { closedLocations } = appConfig;
+    const { closedLocations, recapClosedLocations, nonRecapClosedLocations } = appConfig;
+    // const isRecap = LibraryItem.mapItem(item).isRecap;
```

Comment:
Should this line and line 10 be removed?

Reply:
Good catch